Patches/python.diff

```diff
@@ -211,50 +205,9 @@ diff -r 8667c26e2bec Objects/codeobject.c
  /* Use co_lnotab to compute the line number from a bytecode index, addrq.  See
     lnotab_notes.txt for the details of the lnotab representation.
  */
-diff -r 8667c26e2bec Objects/object.c
---- a/Objects/object.c	Mon Jun 15 09:11:37 2015 -0700
-+++ b/Objects/object.c	Mon Aug 31 14:44:51 2015 -0700
-@@ -1787,12 +1787,12 @@
- _Py_PrintReferences(FILE *fp)
- {
-     PyObject *op;
--    fprintf(fp, "Remaining objects:\n");
-+    fprintf(fp, "Remaining objects:\r\n");
```

Comment:
This is another case of cleaning up the patch file and backing out changes which were made for debugging.  having the new lines is a little annoying on windows but it's how it was before I changed it.  While your suggestion would be good I don't really want to clutter up our patch file with non-JIT related changes, so I'm just trying to eliminate the ones which slipped in